### PR TITLE
Helm chart react changes for the new runner image.

### PR DESCRIPTION
**Pull request by TingluoHuang**
Reaction work to https://github.com/actions/runner/pull/2469

### CODE REVIEW

File: adrs/2022-10-17-runner-image.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 
 **Status**: Done
 
+# Breaking Changes
+
+To make customers feel less fraction when mismatched use this runner image and the GitHub Hosted ubuntu-latest image for their workflow jobs,  
+we are making the following change to align the runner image close to the Hosted ubuntu runner.
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
We aim to provide an similar experience (as close as possible) between self-hosted and GitHub-hosted runners. To achieve this, we are making the following changes to align our self-hosted runner container image with the Ubuntu runners managed by GitHub.
```

**Review by Link- — CHANGES_REQUESTED** [review]

File: adrs/2022-10-17-runner-image.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+Here are the changes:
+- We create a USER `runner(1001)` and a GROUP `docker(123)`
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
- We created a USER `runner(1001)` and a GROUP `docker(123)`
```

File: adrs/2022-10-17-runner-image.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
...
+- `sudo` will be installed on the image and the `runner` will be a passwordless sudoer.
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
- `sudo` has been on the image and the `runner` will be a passwordless sudoer.
```

File: adrs/2022-10-17-runner-image.md [hunk 4]
```diff
@@ -3,6 +3,18 @@
...
+- The runner binary will be placed under `/home/runner/run.sh`
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
- The runner binary was placed placed under `/home/runner/` and launched using `/home/runner/run.sh`
```

File: adrs/2022-10-17-runner-image.md [hunk 5]
```diff
@@ -3,6 +3,18 @@
...
+- The runner's work directory will be `/home/runner/_work`
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
- The runner's work directory is `/home/runner/_work`
```

File: adrs/2022-10-17-runner-image.md [hunk 6]
```diff
@@ -3,6 +3,18 @@
...
+- `$HOME` will point to `/home/runner`
+- The docker image user will be the `runner(1001)`
```
Comment (Link-) [inline adrs/2022-10-17-runner-image.md]:
```suggestion
- The container image user will be the `runner(1001)`
```

**Review by Link- — COMMENTED** [review]
Thank you for fixing this! Left a few minor comments

File: adrs/2022-10-17-runner-image.md [hunk 7]
```diff
@@ -3,6 +3,19 @@
 
 **Status**: Done
 
+# Breaking Changes
```
Comment (nikola-jokic) [inline adrs/2022-10-17-runner-image.md:6]: This ADR is really outdated now... 
Should we make another one and mark this one as outdated?

File: charts/gha-runner-scale-set/values.yaml [hunk 8]
```diff
@@ -50,7 +50,7 @@ template:
     containers:
     - name: runner
       image: ghcr.io/actions/actions-runner:latest
-      command: ["/actions-runner/run.sh"]
+      command: ["/home/runner/run.sh"]
```
Comment (nikola-jokic) [inline charts/gha-runner-scale-set/values.yaml:77]:
```suggestion

```
After entrypoint.sh gets merged and runne rimage is published: https://github.com/actions/actions-runner-controller/pull/2337

**Review by Link- — APPROVED** [review]

**Review by rentziass — APPROVED** [review]

**Pull request merged**